admin/src/scenes/missions/edit.js

```diff
@@ -379,14 +397,54 @@ export default (props) => {
                 </Col>
               </Row>
             </Box>
+            {defaultValue && user.role === "admin" ? (
```

Comment:
```suggestion
            {defaultValue && user.role === "admin" && structures?.length ? (
```

bloquer l'affichage de ce composant si on n'a pas les structures